src/cpp/session/resources/grid/dtviewer.js

```diff
@@ -1193,6 +1208,10 @@ var addResizeHandlers = function(ele) {
          initColWidth = $("#data_cols th:nth-child(" + col + ")").width();
          initTableWidth = $("#rsGridData").width();
          boundsExceeded = 0;
+
+         if (origColWidths[col] == null)
```

Comment:
I think this only works because `undefined == null` is technically true in JS. Do we ever actually expect a null value here? (if not I think we should check for undefined instead)